### DISCUSSION TITLE
Global environment variables

### DISCUSSION THREAD

**Discussion by AlanSandovalDev** [usebruno/bruno#883] [Q&A] [answered]
When I create an environment and add variables, they work only for the current collection I'm working on. If I select a request from another collection, it automatically changes to "No environment" and the environment I created previously doesn't appear. Does that mean that the environments are collection-based but not global?
I have a case that I use in Postman where I have a collection with requests to obtain a token from an authorization server, and I save this token in a global variable and use it in other collections. I cannot replicate the same in Bruno because of what I explained before. 
Is there a way to use variables that work globally (between all collections)?

**Comment by tsteckenborn** [comment, marked as answer]
I think you should read through the issue #115 where the background of that is already explained. If I'm not mistaken your request / question is basically what he's talking about in this [reply](https://github.com/usebruno/bruno/issues/115#issuecomment-1740928789).

**Comment by AlanSandovalDev** [comment]
Thank you, that is basically my scenario.
